db/migrate/20191202190956_add_bat_points_ever_on_to_publishers.rb

```diff
@@ -0,0 +1,5 @@
+class AddBatPointsEverOnToPublishers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :publishers, :bat_points_ever_on, :boolean, default: false
```

Comment:
I think I prefer the name `bat_points_enabled`, what do you think?

Reply:
My impression is that `bat_points_enabled` is a user-defined setting. `ever_on` or `ever_enabled`  is a setting that can only go from false to true but never true to false